nested/none-search.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "moodleCommon": {
+            "metadata": {
+                "description": "Common Moodle values"
+            },
+            "type": "object"
+        }
+    },
+    "resources": [
+    ],
+    "variables": {
+        "documentation01": "This sub-template that represents no moodle global search is activated."
+    }
+}
```

Comment:
Are you sure this template works when 'none-search' is chosen? The `[reference('searchTemplate').outputs.azureSearchKey.value]` in the main template will fail in that case, no? Could you let us know if you tested all three cases of none-search, azure-search, and elastic-search?

Reply:
OK, I had to verify this myself, and unexpectedly, the templates work without an output section here. This is strange (unexpected ARM behavior to me), but maybe it's just another ARM quirk. So consider this as resolved, but it would have been nice if there were any response to this comment.

Reply:
the three search scenarios have been tested and the updated templates will work with all three options.